pyscf/df/df_jk.py

```diff
@@ -260,8 +260,8 @@ def get_jk(dfobj, dm, hermi=1, with_j=True, with_k=True, direct_scf_tol=1e-13):
 
     if not with_k:
         for eri1 in dfobj.loop():
-            rho = numpy.einsum('ix,px->ip', dmtril, eri1)
-            vj += numpy.einsum('ip,px->ix', rho, eri1)
+            vj += numpy.matmul(numpy.matmul(dmtril, eri1.T), eri1)
```

Comment:
Thanks for the suggestion! I personally love the `@` notation but didn't use it here because
- I couldn't find examples of `@` in pyscf and didn't want to be the first
- There are a lot of matrix multiply functions in use (`lib.dot`, `np.dot`, `lib.ddot`) so I thought it would be useful to indicate explicitly which one is being called

To make it more readable, I could split that line or clean it up with functools.reduce

Reply:
Many `np.dot`s were implemented before python2.7 was retired. Any np.dot can be replaced by `@`. But as long as they work, it's okay to leave np.dot there.